pageserver/src/tenant/storage_layer/split_writer.rs

```diff
@@ -481,7 +481,7 @@ mod tests {
 
     #[tokio::test]
     async fn write_split_discard() {
-        write_split_helper("split_writer_write_split_discard", false).await;
+        write_split_helper("split_writer_write_split_discard", true).await;
     }
 
     async fn write_split_helper(harness_name: &'static str, discard: bool) {
```

Comment:
This is a bit hard to review in isolation without a docstring for this function (it should explain what `discard` is meant to mean)